### PR TITLE
Sudoku via pen

### PR DESCRIPTION
**Pull request by Robertha89**
image

### CODE REVIEW

File: code/modules/paperwork/pen.dm [hunk 1]
```diff
@@ -24,6 +24,9 @@
 	var/colour = "black"	//what colour the ink is!
 	pressure_resistance = 2
 
+/obj/item/weapon/pen/suicide_act(mob/user)
+		viewers(user) << "<span class='suicide'>[user] starts scribbling over \himself with the [src.name]! It looks like \he's trying to commit sudoku.</span>"
```
Comment (Markolie) [inline code/modules/paperwork/pen.dm]: Use tabs instead of spaces please.

**Comment by TheDZD** [comment]
:+1: for comedic value.

File: code/modules/paperwork/pen.dm (hunk 1 above)
Comment (Robertha89) [inline code/modules/paperwork/pen.dm]: those are tabs, two of them?

Comment (TheDZD) [inline code/modules/paperwork/pen.dm]: Yeah, these are tabs. It wouldn't compile otherwise, BYOND is pretty finicky about whitespace.

Comment (tigercat2000) [inline code/modules/paperwork/pen.dm]: actually it can compile with spaces

Comment (Markolie) [inline code/modules/paperwork/pen.dm]: Well, make it one tab then.

**Comment by TheDZD** [comment]
I, uh, what the heck? Why did that compile earlier and not now?

**Comment by tigercat2000** [comment]
Byond doesn't actually care about spacing or tabbing.
It cares about consistancy.
So it will allow bullshit spacing/tabbing.
But not inconsistant spacing/tabbing.

**Comment by KasparoVy** [comment]
LMFAO, absolutely hilarious.

**Comment by Aurorablade** [comment]
WHEN INTERNAL AFFAIRS AGENTS CANNOT TAKE IT ANYMORE

File: code/modules/paperwork/pen.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/modules/paperwork/pen.dm]: Yup, these both need only a single tab, not two.

**Comment by TheBeoni** [comment]
Yes please

**Comment by KasparoVy** [comment]
Holy shit that's violent.

**Pull request merged**